lib/plugins/create/templates/aws-fsharp/build.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+#build handlers
+dotnet restore
+dotnet publish -c release
+
+#install zip
+apt-get -qq update
+apt-get -qq -y install zip
+
+#create deployment package
+pushd bin/Release/netcoreapp1.0/publish
```

Comment:
By default it's the same name as the folder you're running the script from.  Override with the `--output-package bin/Release/netcoreapp1.0/Package.zip` flag, e.g:

```
dotnet lambda package --configuration Release --framework netcoreapp1.0  --output-package bin/Release/netcoreapp1.0/Package.zip
```